src/material-experimental/theming/_custom-tokens.scss

```diff
@@ -1083,6 +1083,29 @@
   ));
 }
 
+/// Generates custom tokens for the mat-slide-toggle.
+/// @param {Map} $systems The MDC system tokens
+/// @param {Boolean} $exclude-hardcoded Whether to exclude hardcoded token values
+/// @return {Map} A set of custom tokens for the mat-slide-toggle
+@function switch($systems, $exclude-hardcoded) {
+  @return ((
+    unselected-handle-width: 16px,
+    unselected-handle-height: 16px,
+    selected-handle-width: 24px,
```

Comment:
For my own curiosity: does the M3 slide toggle really have a differently-sized handle depending on the selected state? I would imagine that would cause it to jump around.

Reply:
Yep lol. It's not too bad because of the transition, but I definitely found it strange at first as well
https://m3.material.io/components/switch/specs#34f9a158-a2b3-4c15-841b-578978126788

Reply:
They even have a base state with 20px but from what I can tell, that state is not reachable